haskell/gtk-gui/README.md

```diff
@@ -0,0 +1,45 @@
+# Creating a GUI application in Haskell with GTK+
+
+## Description
+
+This project serves as a companion codebase for Stack Builder's blog post ["Creating a GUI application in Haskell."](https://www.stackbuilders.com/blog/gui-application/)
+
+## Requirements
+
+- Stack 2.11.1
+- Cabal 3.10.2.1
```

Comment:
we don't need this, it's enough with the stack version, that will take care of the rest.